sql/sql_component.cc

```diff
@@ -121,6 +121,22 @@ Deployed_components::Deployed_components(const std::string program_name)
     components_.clear();
   }
 }
+#ifdef XTRABACKUP
+Deployed_components::Deployed_components(const std::string program_name, std::string component)
```

Comment:
I think second parameter should be named as component_names? and it can be a "," separated list. May be just provide an example.
Do we ever load multiple components during PXB online/offline init?